hyperparameter_tuning/mxnet_gluon_cifar10_random_search/hyperparameter_tuning_mxnet_gluon_cifar10_random_search.ipynb

```diff
@@ -41,6 +41,17 @@
     "Specify the IAM role for permission to access the dataset in S3 and SageMaker functionality."
```

Comment:
<p>Could you add a comment as a text above the cell? Right now it states 'specify the IAM role ... ` and then the cell with installation of scikit-image, which is a bit unexpected. IMO would look better like this: </p><p> </p><p> Specify IAM role</p><p><code>the iam role cell</code>  </p><p> </p><p>install scikit-image package </p><p><code>installation cell</code></p><p> </p><p>... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>